Content.Shared/CCVar/CCVars.cs

```diff
@@ -397,7 +387,23 @@ public sealed class CCVars : CVars
         public static readonly CVarDef<string> DiscordRoundEndRoleWebhook =
             CVarDef.Create("discord.round_end_role", string.Empty, CVar.SERVERONLY);
 
+        /// <summary>
+        ///     Discord bot token for link stuff.
+        /// </summary>
```

Comment:
This entire file needs better comments indicating that these various CVars you added are linked and need each other to be configured.